components/landlord-setup/landlord-setup-flow.tsx

```
@@ -0,0 +1,156 @@
+	petRestrictions?: string;
+	utilitiesIncluded?: string[];
+	[key: string]: any;
+}
+
+interface ManagementData {
+	contactDisplay?: string;
+	applicationProcess?: string;
+	screeningPreferences?: string[];
+	communicationPreferences?: string[];
+	leaseSigningPreference?: string;
+	[key: string]: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)